templates/zerver/integrations.html

```diff
@@ -1009,6 +1008,99 @@
 
     </div>
 
+    <div id="google-calendar" class="integration-instructions">
+
+        <p>
+        Get Google Calendar reminders in Zulip! This is a great way to see your reminders directly in
+        your Zulip feed.
+        </p>
+
+        <p>
+        First download and install our <a href="/api">Python Bindings and example scripts</a>.
+        This bot should be set up on a trusted machine, because your API key is visible to local users
+        through the command line or config file.
+        </p>
+
+        <p>
+        Next, follow the instructions for <b>Step 1</b> at <a href="https://developers.google.com/
+        google-apps/calendar/quickstart/python">this link</a> to get a <code>client_secret</code>
+        file. Save this file to your <code>/~</code> directory instead of your working directory.
+        </p>
+
+        <p>
+        Next, install the latest Google API Client for Python by following the instructions on
+        the <a href="https://developers.google.com/api-client-library/python/start/installation">
+        Google Website</a>.
+        </p>
+
+        <p>
+        Then go to your <b>Zulip Settings</b> by clicking on the cog in the top right corner,
+        and then clicking on <b>Settings</b>.
+        </p>
+
+        <p>
+        Click on the tab that's labeled <b>Your Bots</b> and click on <b>Show/change your API
+        key</b>. Enter your password if prompted, and download the <code>.zuliprc</code> file. Save
+        this file to your <code>~/</code> folder.
+        </p>
+
+        <p><img class="screenshot" src="/static/images/integrations/google/calendar/001.png" /></p>
+
+        <p>
+        Run the <code>get-google-credentials</code> with this command:
+        <pre>
+        python /usr/local/share/zulip/integrations/google/get-google-credentials
+        </pre>
```

Comment:
You should get rid of the whitespace around the <pre> entry to get rid of the extra blank lines in the docs.